policy/release/tasks.rego

```diff
@@ -135,14 +135,30 @@ warn contains result if {
 #   - tasks.pipeline_has_tasks
 #
 warn contains result if {
-	some required_task in _missing_tasks(latest_required_tasks)
-
-	# If the required_task is also part of the current_required_tasks, do
-	# not proceed with a warning since that's clearly a violation.
-	not required_task in current_required_tasks
-	result := lib.result_helper_with_term(rego.metadata.chain(), [_format_missing(required_task, true)], required_task)
+	some required_task in _missing_tasks(tkn.get_tasks(latest_required_tasks))
+	# # If the required_task is also part of the current_required_tasks, do
+	# # not proceed with a warning since that's clearly a violation.
+	not required_task in tkn.get_tasks(current_required_tasks)
+	result := lib.result_helper_with_term(rego.metadata.chain(), [_format_missing(required_task, true), tkn.get_effective_on(latest_required_tasks)], required_task)
 }
 
+# deny contains result if {
+# 	not latest_required_tasks.tasks
+# }
+
+# deny contains result if {
+# 	not latest_required_tasks.effective_on
+# }
+
+# deny contains result if {
+# 	not current_required_tasks.tasks
+# }
+
+# deny contains result if {
+# 	not current_required_tasks.effective_on
+# }
```

Comment:
Is this intentionally still here? Maybe take it out unless there's a good reason for it.